encoderfile-core/src/common/model_config.rs

```diff
@@ -0,0 +1,87 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ModelConfig {
+    pub model_type: String,
+    pub pad_token_id: u32,
+    pub num_labels: Option<usize>,
```

Comment:
I suspect this is done for classification? Does it make sense to group class-related items under a `classification` key?